### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by fmrsabino**
- Remove docker development section. If possible this should be done in an environment where python and Django can be run locally (to reduce the scope of the document)

File preview: https://github.com/gnosis/safe-config-service/blob/update-readme/README.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/gnosis/safe-config-service/badge.svg)](https://coveralls.io/github/gnosis/safe-config-service)
 
-The `safe-config-service` is a service that provides configuration information in the context of the Safe clients environment (eg.: list of available safe apps).
+The `safe-config-service` is a service that provides configuration information in the context of the Safe clients environment (eg.: list of available safe apps, or chain metadata).
```
Comment (jpalvarezl) [inline README.md:5]: nitpick: I would say `and` chain metadata

File: README.md [hunk 2]
```diff
@@ -106,8 +75,8 @@ pip install -r requirements-dev.txt
 
 ### Testing
 
-Pytest is used to run the available tests in the project. Some of these tests validate the integration with the database
-so having one running is required. From the project root:
+Pytest is used to run the available tests in the project. **Some of these tests validate the integration with the database
```
Comment (jpalvarezl) [inline README.md:78]: Seems like you didn't close the `**`

**Review by jpalvarezl — APPROVED** [review]
Just a couple of small comments :)

File: README.md (hunk 2 above)
Comment (jpalvarezl) [reply README.md:78]: Ah never mind this comment, I just noticed where you closed it.

File: README.md (hunk 1 above)
Comment (fmrsabino) [reply README.md]: e3e4c27

**Review by jpalvarezl — APPROVED** [review]

**Pull request merged**